src/ldp/permissions/SparqlPatchPermissionsExtractor.ts

```diff
@@ -0,0 +1,58 @@
+import { Algebra } from 'sparqlalgebrajs';
+import { Operation } from '../operations/Operation';
+import { PermissionSet } from './PermissionSet';
+import { PermissionsExtractor } from './PermissionsExtractor';
+import { Representation } from '../representation/Representation';
+import { SparqlUpdatePatch } from '../http/SparqlUpdatePatch';
+import { UnsupportedHttpError } from '../../util/errors/UnsupportedHttpError';
+
+/**
+ * Generates permissions for a SPARQL DELETE/INSERT PATCH.
```

Comment:
```suggestion
 * Generates permissions for a SPARQL DELETE/INSERT patch.
```

(just as to not confuse with the HTTP method)